pkgs/top-level/ocaml-packages.nix

```diff
@@ -58,28 +58,34 @@ let
 
     awa-mirage = callPackage ../development/ocaml-modules/awa/mirage.nix { };
 
-    base64 = callPackage ../development/ocaml-modules/base64 { };
-
     bap = callPackage ../development/ocaml-modules/bap {
       inherit (pkgs.llvmPackages) llvm;
     };
 
+    base64 = callPackage ../development/ocaml-modules/base64 { };
+
     batteries = callPackage ../development/ocaml-modules/batteries { };
 
+    benchmark = callPackage ../development/ocaml-modules/benchmark { };
+
     bheap = callPackage ../development/ocaml-modules/bheap { };
 
     bigarray-compat = callPackage ../development/ocaml-modules/bigarray-compat { };
 
     bigarray-overlap = callPackage ../development/ocaml-modules/bigarray-overlap { };
 
-    bigstringaf = callPackage ../development/ocaml-modules/bigstringaf { };
+    biniou = callPackage ../development/ocaml-modules/biniou { };
```

Comment:
`biniou` is out of order, as it is before `bigstring` here and n should come after g.

Reply:
Thanks!